src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] arr = new Figure[6];
```

Comment:
use constant

Reply:
Can you explain to me why we need to use an interface here if we use it in literally every class, i.e. we could just make an abstract class?